packages/eui/src/components/popover/input_popover.tsx

```diff
@@ -190,15 +190,18 @@ export const EuiInputPopover: FunctionComponent<EuiInputPopoverProps> = ({
         closePopover();
       };
 
-      window.addEventListener('scroll', closePopoverOnScroll, {
-        passive: true, // for better performance as we won't call preventDefault
-        capture: true, // scroll events don't bubble, they must be captured instead
-      });
+      const timeoutId = setTimeout(() => {
```

Comment:
Let's restore the comment that was here as well (replacing "Firefox" with "Kibana Cypress tests", since that's why we're restoring the timeout), so that the next dev knows why the timeout is here

```suggestion
      // Kibana Cypress tests trigger a scroll event in many common situations when the options list div is appended
      // to the DOM; in testing it was always within 100ms, but setting a timeout here for 500ms to be safe 
      const timeoutId = setTimeout(() => {
```

Reply:
Updated in this [commit](https://github.com/elastic/eui/pull/7760/commits/e1528553ba1564940924e9ba4e5ee42c742d8302)